drivers/timer/nrf_grtc_timer.c

```diff
@@ -518,7 +521,32 @@ static int sys_clock_driver_init(void)
 #if defined(CONFIG_NRF_GRTC_ALWAYS_ON)
 	nrfx_grtc_active_request_set(true);
 #endif
+
+#if DT_PROP(GRTC_NODE, clkout_32k)
+	nrfy_grtc_clkout_set(NRF_GRTC, NRF_GRTC_CLKOUT_32K, true);
+#endif
+
+#if DT_NODE_HAS_PROP(GRTC_NODE, clkout_fast_frequency)
+#if DT_NODE_HAS_PROP(HFCLK_NODE, clock_frequency) &&                                               \
+	DT_PROP(GRTC_NODE, clkout_fast_frequency) > (DT_PROP(HFCLK_NODE, clock_frequency) / 2)
+#error "Invalid frequency value for fast clock output."
+#endif
+	uint32_t base_frequency = DT_PROP(HFCLK_NODE, clock_frequency);
+	uint32_t requested_frequency = DT_PROP(GRTC_NODE, clkout_fast_frequency);
+	uint32_t grtc_div = base_frequency / (requested_frequency * 2);
+
+	nrfy_grtc_clkout_set(NRF_GRTC, NRF_GRTC_CLKOUT_FAST, true);
+	nrfy_grtc_clkout_divider_set(NRF_GRTC, (uint8_t)grtc_div);
```

Comment:
Is it okay to set divider after enabling `clkout`?

Reply:
switched the order, however it doesn't matter